src/SIL.XForge.Scripture/ClientApp/src/app/home/home.component.html

```diff
@@ -16,3 +16,5 @@ <h1 *ngIf="name">Hello, {{name}}!</h1>
 <button mat-stroked-button (click)="logoff()">
   Logout
 </button>
+
+<button mat-stroked-button (click)="changePassword()">Change Password</button>
```

Comment:
You can use `[routerLink]="['/change-password']"` in the `button` element to route to the change password component. You don't need to inject `Router` into the `HomeComponent` class.